src/gui/CGALWorker.cc

```diff
@@ -36,6 +37,11 @@ void CGALWorker::work()
   try {
     GeometryEvaluator evaluator(*this->tree);
     root_geom = evaluator.evaluateGeometry(*this->tree->root(), true);
+
+    if (auto manifold = std::dynamic_pointer_cast<const ManifoldGeometry>(root_geom)) {
+      if (manifold->getManifold().Status() != manifold::Manifold::Error::NoError)
```

Comment:
Yes, I can add it here and to the upstream documentation. The rationale is that it doesn't make sense to avoid evaluation when you check the status, as evaluations can cause issues and the point of checking status is to check for those issues :)

Btw, I guess you forgot to click submit reviews previously? GitHub somehow said the comment is 3 days old, even though the entire review is from a few minutes ago.

Reply:
Yeah, was writing on my phone and forgot to press the submit button - just realized when you requested a review :)